plugwise/smile.py

```diff
@@ -54,7 +54,7 @@ def update_for_cooling(self, devices: dict[str, DeviceData]) -> None:
         """Helper-function for adding/updating various cooling-related values."""
         for _, device in devices.items():
             # For Anna + cooling, modify cooling_state based on provided info by Plugwise
-            if self.smile_name == "Anna":
+            if self.smile_name == "Smile":
```

Comment:
Another refinement, we could CONST the name (for future renaming)

Reply:
Nah, the CONSTs just result in more lines and we don't save any lines using them. So let's not.